### ISSUE TITLE
"bounds.js:10 Uncaught TypeError: Cannot read property 'length' of undefined" error

### ISSUE THREAD

**Issue by nicoleta2540** [marmelab/EventDrops#246]
Hello,

I have this error in version 1.1.0 from https://unpkg.com/

```
bounds.js:10 Uncaught TypeError: Cannot read property 'length' of undefined
    at bounds.js:10
    at ut.call (d3.v4.min.js:2)
    at index.js:114
    at ut.call (d3.v4.min.js:2)
    at g (index.js:73)
    at ut.call (d3.v4.min.js:2)
    at init_ActivityChart (managdevices.js:740)
    at Object.complete (managdevices.js:2916)
    at i (jquery.js:2)
    at Object.fireWith (jquery.js:2)
```

How can this be solved?

Thank you!

**Comment by jpetitcolas** [comment]
We can indeed improve the error message. Can you reproduce the issue on the [JSFiddle](https://jsfiddle.net/6rqxusw5/9/)? This way, we will be able to help you spotting the issue in your code. :)

**Comment by mrmelson** [comment]
I'm getting this same error & I believe i can help with recreating it. 

My code is very similar to the demo code. And, I can recreate the error by calling eventdrops twice in a row. E.G. Repeat this block of code 2x at the bottom of demo.js:
```
d3
    .select('#eventdrops-demo')
    .data([repositoriesData])
    .call(chart);
```

**Comment by mrmelson** [comment]
This issue appears to be related to this [question on Stackoverflow](https://stackoverflow.com/questions/50615737/add-and-remove-lines-from-eventdrops-visualization).

I think it may be as a result of attempting to add/change/remove data after creating the initial chart. This is what I'm trying to do, anyway. 

Thanks!

**Comment by nicoleta2540** [comment]
@mrmelson you are right. I could fix the problem in the code. I was trying to do same thing as you, add data after creating the initial chart.

Thanks a lot!

**Comment by fzaninotto** [comment]
Fixed by #262
